packages/neuron-wallet/src/listeners/address.ts

```diff
@@ -1,12 +1,28 @@
+import { remote } from 'electron'
+import { ReplaySubject } from 'rxjs'
+import { bufferTime } from 'rxjs/operators'
 import AddressesUsedSubject from '../models/subjects/addresses-used-subject'
 import AddressService from '../services/addresses'
 import WalletService from '../services/wallets'
 import { AccountExtendedPublicKey } from '../models/keys/key'
 
+const isRenderer = process && process.type === 'renderer'
+const addressesUsedSubject = isRenderer
+  ? remote.require('./models/subjects/addresses-used-subject').default.getSubject()
+  : AddressesUsedSubject.getSubject()
+
+// pipe not working directly
+const bridge = new ReplaySubject<string[]>(1000)
+addressesUsedSubject.subscribe((addresses: string[]) => {
+  bridge.next(addresses)
+})
+
 // update txCount when addresses used
 export const register = () => {
-  AddressesUsedSubject.getSubject().subscribe(async (addresses: string[]) => {
-    const addrs = await AddressService.updateTxCountAndBalances(addresses)
+  bridge.pipe(bufferTime(1000)).subscribe(async (addressesList: string[][]) => {
+    const addresses = addressesList.reduce((acc, val) => acc.concat(val), [])
```

Comment:
```suggestion
    const addresses = addressesList.flat()
```

Reply:
`flat()` should enable module in tsconfig, and it may cause new problems